src/components/common/icons/base.tsx

```diff
@@ -73,13 +75,16 @@ export default class Base extends Component {
     const fillStyle =
       Array.isArray(colors) && totalColor && getStyleClassFromColor(totalColor, colors);
 
+    const nop = () => {};
```

Comment:
Sorry, I mean outside of the component scope. in render it will start gets instantiated every render.

Can we put it in defaultProps?


```js
 static defaultProps = {
    height: null,
    onClick: nop
}

```